source/simulator/core.cc

```diff
@@ -1365,6 +1403,95 @@ namespace aspect
 
 
   template <int dim>
+  class VectorFunctionFromVectorFunctionObject : public Function<dim>
```

Comment:
Actually, this class already exists in deal ii, doesn't it? Or am I confusing it with one of the other similar classes?